<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="p1">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>221</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 1</string>
    </property>
    <widget class="QLabel" name="p1Name">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>25</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Player 1 Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dollarLabelp1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DOLLARS: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p1Money">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>$100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="posLabelp1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>POSITION:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p1Pos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="powerLabelp1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>POWERS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="p1Power0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p1Power1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p1Power2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="p2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>-1</y>
      <width>221</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 2</string>
    </property>
    <widget class="QLabel" name="posLabelp2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>POSITION:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dollarLabelp2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DOLLARS: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p2Pos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="p2Power0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p2Power1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p2Power2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Power0</string>
     </property>
    </widget>
    <widget class="QLabel" name="p2Money">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>$100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="powerLabelp2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>POWERS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="p2Name">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>25</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminal</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Player 2 Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>711</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainGrid"/>
   </widget>
   <widget class="QLabel" name="Dice">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DICE: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rollDice">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ROLL</string>
    </property>
   </widget>
   <widget class="QLabel" name="Turn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TURN:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COMP SCI LIFE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
